packages/components/managed-data-table/src/data-table-settings/droppable-panel/draggable-tag/draggable-tag.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
+import { Draggable } from 'react-beautiful-dnd';
+import { DragIcon } from '@commercetools-uikit/icons';
+import Spacings from '@commercetools-uikit/spacings';
+import Tag from '@commercetools-uikit/tag';
+import DraggingContainer from './draggable-tag.styles';
+
+const portal = document.createElement('div');
+portal.classList.add('dnd-portal');
+document.body.appendChild(portal);
```

Comment:
What is this?? 😳 Why is it defined here in the module and not in the component?

Reply:
@NickDevG can you clarify this point?

Reply:
Honestly this was on the original code and I didn't look into it. After removing it everything is working as it used to.